src/GUI.cpp

```diff
@@ -428,6 +683,50 @@ static void Display_WindowOptions(HealTableOptions& pHealingOptions, HealWindowC
 			ImGuiEx::AddTooltipToLastItem("Show a colored bar under each entry signifying what the value of\n"
 				"that entry is in proportion to the largest entry");
 
+			if (ImGuiEx::SmallCheckBox("use subgroup for bar colour", &pContext.UseSubgroupForBarColour) == true && pContext.UseSubgroupForBarColour == true)
+			{
+				// Mutually exclusive with "use profession for bar colour"
+				pContext.UseProfessionForBarColour = false;
+			}
+			if (ImGuiEx::SmallCheckBox("use profession for bar colour", &pContext.UseProfessionForBarColour) == true && pContext.UseProfessionForBarColour == true)
+			{
+				// Mutually exclusive with "use subgroup for bar colour"
+				pContext.UseSubgroupForBarColour = false;
+			}
+			ImGuiEx::SmallCheckBox("index numbers", &pContext.IndexNumbers);
+			ImGuiEx::SmallCheckBox("profession text", &pContext.ProfessionText);
+			ImGuiEx::SmallCheckBox("profession icons", &pContext.ProfessionIcons);
+			ImGuiEx::SmallCheckBox("replace player with account name", &pContext.ReplacePlayerWithAccountName);
+			if(ImGuiEx::SmallCheckBox("use profession for name colour", &pContext.UseProfessionForNameColour) == true && pContext.UseProfessionForNameColour == true)
+			{
+				// Mutually exclusive with "use subgroup for name colour"
+				pContext.UseSubgroupForNameColour = false;
+			}
+			if (ImGuiEx::SmallCheckBox("use subgroup for name colour", &pContext.UseSubgroupForNameColour) == true && pContext.UseSubgroupForNameColour == true)
+			{
+				// Mutually exclusive with "use profession for name colour"
+				pContext.UseProfessionForNameColour = false;
+			}
+			if (ImGuiEx::SmallCheckBox("self on top", &pContext.SelfOnTop) == true && pContext.SelfOnTop == true)
+			{
```

Comment:
should this be hidden / grayed out for non-agent windows? It makes no sense to set this for skills e.g. right?

Reply:
Many Display options make no sense for anything that is not DataSource::PeersOutgoing , at least in the way those options are implemented. Any preference about how to deal with that ?